ansible/roles/kubernetes-common/defaults/main.yml

```diff
@@ -16,6 +16,22 @@ kubernetes_common_kubeadm_config:
   controlPlaneEndpoint: "{{ kubernetes_common_api_fqdn }}"
   apiServerExtraArgs:
     "endpoint-reconciler-type": "lease"
+    "audit-policy-file": "/etc/kubernetes/apiserver-audit-policy.yaml"
+    "audit-log-path": "/var/log/kubernetes/kube-apiserver-audit.log"
+    "audit-log-maxage": "30" # days
+    "audit-log-maxsize": "100" # megabytes
+    "audit-log-maxbackup": "1"
+  apiServerExtraVolumes:
+  - name: audit-policy
+    hostPath: /etc/kubernetes/apiserver-audit-policy.yaml
+    mountPath: /etc/kubernetes/apiserver-audit-policy.yaml
+    writable: false
+    pathType: File
+  - name: audit-logs
+    hostPath: /var/log/kubernetes
+    mountPath: /var/log/kubernetes
```

Comment:
Does this directory always exist? Should be sure that we create it/ensure it exists.

Reply:
Is there a reason not to have log destination set to - (stdout) so that standard docker log rotation and any log aggregation would be used?  If staying with `/var/log/kubernetes`, what log rotation is in place, or what's available to prevent filling the partition?

Reply:
Logs are sent to /var/log/kubernetes/ based on the default fluentd-kubernetes-daemonset configuration:

https://github.com/fluent/fluentd-kubernetes-daemonset/blob/master/docker-image/v1.3/debian-elasticsearch/conf/kubernetes.conf#L171